recipes/neworder/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "neworder" %}
+{% set version = "0.0.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/neworder-{{ version }}.tar.gz
+  sha256: 43eb1fdc8aff099a4a40ae0b1011e7f676780157731c00e03594c531f151c57b
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - pybind11 >=2.5.0
+    - pytest-runner
+    - python
+  run:
+    - python
+    - numpy
+    - pandas
+
+test:
+  imports:
+    - neworder
+  commands:
+    - pip check
+    - pytest
+  requires:
+    - pip
+    - pytest
+    - numpy
+    - pandas
```

Comment:
the only entry needed here is pytest

Reply:
ok thanks will remove and reinstate pytest with verbose flags, as I don't quite get why the logs say no tests are being run:

```
============================= test session starts ==============================
platform linux -- Python 3.6.11, pytest-6.0.2, py-1.9.0, pluggy-0.13.1
rootdir: $SRC_DIR
collected 0 items
```

Reply:
now saying 
`home/conda/staged-recipes/build_artifacts/neworder_1600287643571/test_tmp/run_test.sh: line 7: pip: command not found`
so going to add pip back to requres

Reply:
hmm weird yeah